product_docs/docs/pgd/5/manual/08-using-pgd-cli.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Step 8 - Using PGD CLI
+navTitle: Using PGD CLI
+deepToC: true
+---
+
+## Using PGD CLI
+
+The PGD CLI command is authenticated in the same way as the psql command. Use an account which you have previously configured authentication for used to run it.
+
+!!! Important Setting passwords
```

Comment:
This red warning box is totally throwing me.  If one uses the `enteprisedb` user, won't pgd cli authenticate without needing to set a password?  The [PGD CLI docs ](https://deploy-preview-5104--edb-docs-staging.netlify.app/docs/pgd/latest/cli/installing_cli/)state "To use PGD CLI, you need a user with PGD superuser privileges or equivalent"

Reply:
as a follow-up, I see that "red box" is actually duplicated from the full-coverage CLI docs section.  I'd suggest not duplicating that here.